extension/src/experiments/webview/messages.ts

```diff
@@ -72,11 +68,9 @@ export class WebviewMessages {
     this.hasStages = hasStages
     this.addStage = addStage
     this.selectBranches = selectBranches
-    this.getNumCommits = getNumCommits
     this.update = update
 
     void this.changeHasConfig()
-    void this.changeHasMoreOrLessCommits()
```

Comment:
[F] I found all of this hard to track so move the call out of `WebviewMessages` and into `ExperimentsData`. What this means is that all of the data/transformation logic should be trapped inside of `ExperimentsModel` making the overall flow easier to understand.